### PR TITLE
Add "php5-sqlite" to dependencies

### PR DESCRIPTION
**Pull request by WaLLy3K**
**By submitting this pull request, I confirm the following:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

9

---

When loading Long Term Data > Query Log, this prevents the following message from popping up:

> An unknown error occured while loading the data.

### CODE REVIEW

**Comment by WaLLy3K** [comment]
Given the various names for the package we need (and the potential uncertainty of the package not existing), we might want to consider doing a search for it beforehand with: `apt-cache search --names-only "^php" sqlite | cut -d " " -f1`

What do you think, @DL6ER?

**Comment by DL6ER** [comment]
@WaLLy3K Sounds like a good idea.

**Comment by dschaper** [comment]
Running on Armbian with `php 7` I get the following packages listed:

```
root@nanopineo:/var/log/lighttpd# apt-cache search --names-only "^php" sqlite | cut -d " " -f1
php-sqlite3
php7.0-sqlite3
```

The first is php5, the second is php7.

**Comment by WaLLy3K** [comment]
Ah, so we're going to just have to amend the `--names-only` search to limit it to the users currently installed PHP version. Good catch, @dschaper!

**Comment by dschaper** [comment]
https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L102 This is the check we do during install to see what PHP version is available, would this be of any use for the issue in this thread?

**Comment by WaLLy3K** [comment]
It could work, but I don't know how that'll affect anyone that's wanting to run PHP7 instead of 5.

**Comment by WaLLy3K** [comment]
Currently, it looks like `apt-cache search --names-only "^${phpVer}" sqlite | cut -d " " -f1` will be the easiest way to *ensure* we're getting the right package, since the package name isn't exactly reliable. However, this won't work on the Fedora family. Could anyone confirm what the `apt-cache` equivalent is?

**Comment by PromoFaux** [comment]
Is there a `yum` equivalent? @bcambl may know...

**Comment by bcambl** [comment]
php-pdo is what you are looking for.

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -204,7 +209,7 @@ elif command -v rpm &> /dev/null; then
   PKG_COUNT="${PKG_MANAGER} check-update | egrep '(.i686|.x86|.noarch|.arm|.src)' | wc -l"
   INSTALLER_DEPS=(dialog git iproute net-tools newt procps-ng)
   PIHOLE_DEPS=(bc bind-utils cronie curl dnsmasq findutils nmap-ncat sudo unzip wget)
-  PIHOLE_WEB_DEPS=(lighttpd lighttpd-fastcgi php php-common php-cli)
+  PIHOLE_WEB_DEPS=(lighttpd lighttpd-fastcgi php php-common php-cli php-pdo)
```
Comment (Mcat12) [inline automated install/basic-install.sh]: Is PDO necessary?

**Comment by PromoFaux** [comment]
Please note the three additional commits are from the `development` branch. From a PR that probably should have gone into 3.2 in the first place (cosmetic bugs)

File: automated install/basic-install.sh (hunk 1 above)
Comment (PromoFaux) [inline automated install/basic-install.sh]: Based off of @bcambl 's comment here:
https://github.com/pi-hole/pi-hole/pull/1581#issuecomment-318190684

File: automated install/uninstall.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ if [ -x "$(command -v rpm)" ]; then
 		PKG_MANAGER="yum"
 	fi
 	PKG_REMOVE="${PKG_MANAGER} remove -y"
-	PIHOLE_DEPS=( bind-utils bc dnsmasq lighttpd lighttpd-fastcgi php-common git curl unzip wget findutils )
+	PIHOLE_DEPS=( bind-utils bc dnsmasq lighttpd lighttpd-fastcgi php-common php-sqlite git curl unzip wget findutils )
```
Comment (Mcat12) [inline automated install/uninstall.sh]: I think this should be `php-pdo`?

Comment (PromoFaux) [inline automated install/uninstall.sh]: Well caught. Uninstall script neglected as-per! One sec...

File: automated install/basic-install.sh (hunk 1 above)
Comment (Mcat12) [reply automated install/basic-install.sh]: Verified that (at least on Fedora) it provides the same SQLite library.

**Pull request merged**